README.md

```diff
@@ -9,259 +16,41 @@
 
 ## Introduction
 
-The OCM provider allows Terraform to manage Red Hat OpenShift Service on AWS (ROSA) clusters, machine pools and identity provider.
+The Red Hat OCM provider allows Terraform to manage Red Hat OpenShift Service on AWS (ROSA) clusters, machine pools, and identity provider.
```

Comment:
I think you might be missing an article before "identity provider". If we were to remove the bits about the clusters and machine pools the sentence would read: "The Red Hat OCM provider allows Terraform to manage...identity provider. Should it be either - 1). identity providers or 2). an/the identity provider?